### DISCUSSION TITLE
Does renovate support pnpm v8?

### DISCUSSION THREAD

**Discussion by dwightjack** [renovatebot/renovate#27435] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Hello, I searched the changelogs and this repo, but I cannot find any indication whether renovate has official support for pnpm v8 (I found one about pnpm v7 though).
Can anyone confirm that renovate works with pnpm v8? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Yes, and it's in use in this repo: https://github.com/renovatebot/renovate/blob/67718f15d54b6ce3f54ed63df99394d10f941dcc/package.json#L139
